lib/config/flat-config-schema.js

```diff
@@ -5,6 +5,16 @@
 
 "use strict";
 
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+/*
+ * Note: This can be removed in ESLint v9 because structuredClone is available globally
+ * starting in Node.js v17.
+ */
+const { default: structuredClone } = require("@ungap/structured-clone");
```

Comment:
Looks like this causes problems for wdio, this is an error we're getting on CI browser test:

```
Error in "Linter with FlatConfigArray.verify().Rule Severity.rule should run as warning when set to 1 with a config array"
Error: Key "rules": Key "semi": structuredClone is not a function
    at http://localhost:40387/node_modules/@humanwhocodes/object-schema/src/object-schema.js:281:27
    at Array.reduce (<anonymous>)
    at _ObjectSchema.merge (http://localhost:40387/node_modules/@humanwhocodes/object-schema/src/object-schema.js:268:24)
    at http://localhost:40387/node_modules/@humanwhocodes/config-array/api.js:966:42
    at Array.reduce (<anonymous>)
    at FlatConfigArray.getConfig (http://localhost:40387/node_modules/@humanwhocodes/config-array/api.js:965:39)
    at Linter._verifyWithFlatConfigArray (http://localhost:40387/build/eslint.js:2:61[45](https://github.com/eslint/eslint/actions/runs/6577334057/job/17871071555#step:5:46)5)
    at Linter.verify (http://localhost:40387/build/eslint.js:2:56041)
    at Context.call (http://localhost:40387/home/runner/work/eslint/eslint/tests/lib/linter/linter.js:12136:41)
```

I'm getting the same errors locally, but it works (at least for me locally) when switched to this form:

```suggestion
const structuredClone = require("@ungap/structured-clone").default;
```

Reply:
We can fix it to make things work, but seems like we should ping @christian-bromann to let him know that this is an issue. I can't imagine why destructuring wouldn't work.

Reply:
I can't remember what particular error I experience but this becomes problematic when transforming into ESM, let me take a look